crates/base/src/edge_runtime.rs

```diff
@@ -542,4 +568,12 @@ mod test {
         let data = user_rt.run(unix_stream_rx).await.unwrap();
         assert_eq!(data, EdgeCallResult::HeapLimitReached);
     }
+
+    #[tokio::test]
+    async fn test_oak_60_issue() {
```

Comment:
Need to fix this. 
Test is throwing
```worker thread panicked Uncaught null

called `Result::unwrap()` on an `Err` value: Uncaught null```

which makes no sense because it works with the server, and tests basically replicate the same behavior as a server request. Not sure what's going on. I'll take a look, otherwise i'll comment it